pkg/gitparse/gitparse.go

```diff
@@ -606,6 +612,13 @@ func (c *Parser) FromReader(ctx context.Context, stdOut io.Reader, commitChan ch
 	ctx.Logger().V(2).Info("finished parsing git log.", "total_log_size", totalLogSize)
 }
 
+func (c *Parser) contentWriter() contentWriter {
```

Comment:
There might be a cleaner way to do this, but my brain couldn't think of it. Happy to change this if anyone has a better approach.

Reply:
Yes, thanks you.